sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Release History
+
+## 1.0.0-beta.3 (Unreleased)
```

Comment:
Should this say `1.0.0-beta.2`?

Reply:
Common is an odd ball. We didn't release beta.2 but we do want next release to be consistent with the others, i.e. next should be beta.3 (Unreleased)

Reply:
@vcolin7 JP is trying to sync this repo's version with the other languages.